### PR TITLE
Create quicksort.c

### PR DESCRIPTION
**Pull request by mateuseap**

### CODE REVIEW

File: sorting/quicksort.c [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        s = HoarePartition(vet, l, r); 
+        quickSort(vet, l, s-1);
+        quickSort(vet, s+1, r);
+    }
+}
+
+int main(){
+
+    return 0;
+}
```
Comment (ashwek) [inline sorting/quicksort.c]: quicksort is never called from the main program.

**Comment by ashwek** [comment]
@mateuseap please add comments in English

File: sorting/quicksort.c (hunk 1 above)
Comment (mateuseap) [reply sorting/quicksort.c]: I removed the main function.

**Comment by mateuseap** [comment]
> @mateuseap please add comments in English

I removed my brazilian portuguese comments and I added two little english comments.

**Comment by ashwek** [comment]
@mateuseap thank you for contributing, but we already have an implementation of quick sort. Closing PR.

**Pull request closed without merging**